packages/safe-core-sdk/src/ethereumLibs/Web3Adapter.ts

```diff
@@ -24,7 +26,7 @@ export interface Web3AdapterConfig {
 }
 
 class Web3Adapter implements EthAdapter {
```

Comment:
Adding some context:
`EthAdapter` is an interface implemented by `Web3Adapter` and `EthersAdapter` classes where all the methods are defined in the interface and implemented using Web3 and ethers.js libraries in the 2 classes respectively. This way it doesn't matter if a Dapp uses ethers.js or web3, they will still be able to access the same functionality.

It would be cool if the 2 new methods are included in `EthAdapter` and `EthersAdapter`.

Reply:
done!